apps/src/gamelab/debugger/replay.js

```diff
@@ -0,0 +1,47 @@
+const trackedProps = ['x', 'y', 'tint'];
+const log = [];
+
+export default function wrap(p5) {
+  const original = p5.createSprite;
+  log.length = 0;
+
+  p5.createSprite = function () {
+    const sprite = original.apply(p5, arguments);
+
+    for (let prop of trackedProps) {
+      const descriptor = Object.getOwnPropertyDescriptor(sprite, prop) || {};
+      Object.defineProperty(sprite, prop, {
```

Comment:
This would be a great place to use an [ES6 Proxy](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy) but unfortunately it's [not supported in IE11](https://caniuse.com/#feat=proxy) 😕.